// @clerk/shared/dist/error.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
const require_clerkApiError = require('./errors/clerkApiError.js');
const require_parseError = require('./errors/parseError.js');
const require_clerkError = require('./errors/clerkError.js');
const require_clerkApiResponseError = require('./errors/clerkApiResponseError.js');
const require_missingExpiredTokenError = require('./errors/missingExpiredTokenError.js');
const require_clerkRuntimeError = require('./errors/clerkRuntimeError.js');
const require_clerkOfflineError = require('./errors/clerkOfflineError.js');
const require_errorThrower = require('./errors/errorThrower.js');
const require_emailLinkError = require('./errors/emailLinkError.js');
const require_webAuthNError = require('./errors/webAuthNError.js');
const require_helpers = require('./errors/helpers.js');
const require_globalHookError = require('./errors/globalHookError.js');

exports.ClerkAPIError = require_clerkApiError.ClerkAPIError;
exports.ClerkAPIResponseError = require_clerkApiResponseError.ClerkAPIResponseError;
exports.ClerkError = require_clerkError.ClerkError;
exports.ClerkOfflineError = require_clerkOfflineError.ClerkOfflineError;
exports.ClerkRuntimeError = require_clerkRuntimeError.ClerkRuntimeError;
exports.ClerkWebAuthnError = require_webAuthNError.ClerkWebAuthnError;
exports.EmailLinkError = require_emailLinkError.EmailLinkError;
exports.EmailLinkErrorCode = require_emailLinkError.EmailLinkErrorCode;
exports.EmailLinkErrorCodeStatus = require_emailLinkError.EmailLinkErrorCodeStatus;
exports.MissingExpiredTokenError = require_missingExpiredTokenError.MissingExpiredTokenError;
exports.buildErrorThrower = require_errorThrower.buildErrorThrower;
exports.createClerkGlobalHookError = require_globalHookError.createClerkGlobalHookError;
exports.errorToJSON = require_parseError.errorToJSON;
exports.is429Error = require_helpers.is429Error;
exports.is4xxError = require_helpers.is4xxError;
exports.isCaptchaError = require_helpers.isCaptchaError;
exports.isClerkAPIError = require_clerkApiError.isClerkAPIError;
exports.isClerkAPIResponseError = require_clerkApiResponseError.isClerkAPIResponseError;
exports.isClerkError = require_clerkError.isClerkError;
exports.isClerkRuntimeError = require_clerkRuntimeError.isClerkRuntimeError;
exports.isEmailLinkError = require_helpers.isEmailLinkError;
exports.isKnownError = require_helpers.isKnownError;
exports.isMetamaskError = require_helpers.isMetamaskError;
exports.isNetworkError = require_helpers.isNetworkError;
exports.isPasswordCompromisedError = require_helpers.isPasswordCompromisedError;
exports.isPasswordPwnedError = require_helpers.isPasswordPwnedError;
exports.isReverificationCancelledError = require_helpers.isReverificationCancelledError;
exports.isUnauthenticatedError = require_helpers.isUnauthenticatedError;
exports.isUnauthorizedError = require_helpers.isUnauthorizedError;
exports.isUserLockedError = require_helpers.isUserLockedError;
exports.parseError = require_parseError.parseError;
exports.parseErrors = require_parseError.parseErrors;
