// @clerk/shared/dist/telemetry.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
const require_collector = require('./telemetry/collector.js');
const require_component_mounted = require('./telemetry/events/component-mounted.js');
const require_flow_step = require('./telemetry/events/flow-step.js');
const require_method_called = require('./telemetry/events/method-called.js');
const require_framework_metadata = require('./telemetry/events/framework-metadata.js');
const require_theme_usage = require('./telemetry/events/theme-usage.js');

exports.EVENT_SAMPLING_RATE = require_theme_usage.EVENT_SAMPLING_RATE;
exports.EVENT_THEME_USAGE = require_theme_usage.EVENT_THEME_USAGE;
exports.TelemetryCollector = require_collector.TelemetryCollector;
exports.eventComponentMounted = require_component_mounted.eventComponentMounted;
exports.eventFlowStepMounted = require_flow_step.eventFlowStepMounted;
exports.eventFrameworkMetadata = require_framework_metadata.eventFrameworkMetadata;
exports.eventMethodCalled = require_method_called.eventMethodCalled;
exports.eventPrebuiltComponentMounted = require_component_mounted.eventPrebuiltComponentMounted;
exports.eventPrebuiltComponentOpened = require_component_mounted.eventPrebuiltComponentOpened;
exports.eventThemeUsage = require_theme_usage.eventThemeUsage;
