// @clerk/shared/dist/utils/index.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
const require_runtimeEnvironment = require('./runtimeEnvironment.js');
const require_handleValueOrFn = require('./handleValueOrFn.js');
const require_instance = require('./instance.js');
const require_noop = require('./noop.js');
const require_createDeferredPromise = require('./createDeferredPromise.js');
const require_allSettled = require('./allSettled.js');
const require_logErrorInDevMode = require('./logErrorInDevMode.js');
const require_runIfFunctionOrReturn = require('./runIfFunctionOrReturn.js');
const require_fastDeepMerge = require('./fastDeepMerge.js');
const require_timeLimit = require('./timeLimit.js');

exports.allSettled = require_allSettled.allSettled;
exports.automatedEnvironmentVariables = require_runtimeEnvironment.automatedEnvironmentVariables;
exports.createDeferredPromise = require_createDeferredPromise.createDeferredPromise;
exports.fastDeepMergeAndKeep = require_fastDeepMerge.fastDeepMergeAndKeep;
exports.fastDeepMergeAndReplace = require_fastDeepMerge.fastDeepMergeAndReplace;
exports.handleValueOrFn = require_handleValueOrFn.handleValueOrFn;
exports.isAutomatedEnvironment = require_runtimeEnvironment.isAutomatedEnvironment;
exports.isDevelopmentEnvironment = require_runtimeEnvironment.isDevelopmentEnvironment;
exports.isProductionEnvironment = require_runtimeEnvironment.isProductionEnvironment;
exports.isStaging = require_instance.isStaging;
exports.isTestEnvironment = require_runtimeEnvironment.isTestEnvironment;
exports.logErrorInDevMode = require_logErrorInDevMode.logErrorInDevMode;
exports.noop = require_noop.noop;
exports.runIfFunctionOrReturn = require_runIfFunctionOrReturn.runIfFunctionOrReturn;
exports.timeLimit = require_timeLimit.timeLimit;
